src/index.js

```diff
@@ -1,4 +1,5 @@
 import './css/styles.css';
+import { createGame, refresh, submit } from './api.js';
 
 function initialize() {
   const root = document.getElementById('root');
```

Comment:
- [ ] Kindly try to use the arrow functions instead of the function keyword. Arrow functions provide a simpler and more concise way to define functions. Kindly change the functions to arrow functions.